package.json

```diff
@@ -96,11 +98,14 @@
     "npm": "^10.2.2",
     "prettier": "^3.0.3",
     "rollup": "^2.75.6",
-    "vue": "^3"
+    "vue": "^3.0.0"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "dependencies": {
+    "semver": "^7.7.1"
```

Comment:
@jasonlundien I think this may less than ideal. By listing this here we'll be subjecting everyone who installs `vue-fontawesome` to also grab this. That might be overkill. What do you think? We could write our own small function that gives us what we need instead of using `semver`.